site/src/server.js

```diff
@@ -94,7 +94,12 @@ if (process.env.GITHUB_CLIENT_ID) {
 
 app.use(
 	compression({ threshold: 0 }),
-	serve('static', { setHeaders: res => res.setHeader('Access-Control-Allow-Origin', '*') }),
+	serve('static', {
```

Comment:
This should be `sirv` and not `serve`, no?

Reply:
Yes, thank you. Juggling too many stashes at once